cypress/e2e/hermioneAccount.cy.js

```diff
@@ -0,0 +1,60 @@
+import { faker } from '@faker-js/faker'
+
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
+  const customerLoginBtn = 'Customer Login';
+  const user = 'Hermoine Granger';
+  const depositBtn = 'Deposit';
+  const depositSuccessMesage = 'Deposit Successful';
+  const withdrawlBtn = 'Withdrawl';
+  const withdrawlSuccessMesage = 'Transaction successful';
+  const withdrawBtn = 'Withdraw';
+  const transactionsBtn = 'Transactions';
+  const backBtn = 'Back';
+  const logoutBtn = 'Logout';
+
+  before(() => {
+    cy.visit('https://globalsqa.com/angularJs-protractor/BankingProject/#/login');
+  });
+
+  it('should provide the ability to work with Hermione\'s bank account', () => {
+
+    cy.contains('.btn', customerLoginBtn).click();
+    cy.url().should('contain', 'customer');
+    cy.get('#userSelect').select(user);
+    cy.get('[type="submit"]').click();
+    cy.get('.fontBig').should('contain', user)
+
+    cy.contains('Account Number').next().should('exist');
+    cy.contains('Balance').next().should('exist');
+    cy.contains('Currency').next().should('exist');
+
+    cy.contains('.btn', depositBtn).click();
+    cy.get('[placeholder="amount"]').type('10');
+    cy.contains('.btn-default', depositBtn).click();
+    cy.contains('.error', depositSuccessMesage).should('be.visible');
+    cy.contains('Balance').should('contain', '100');
```

Comment:
```suggestion
    cy.contains('Balance').should('contain', '100');
```
check on ui mode what does this code checks. It passes because your account number 1001 contains substring 100